src/MassTransit/Middleware/ConsumerMessageFilter.cs

```diff
@@ -3,7 +3,8 @@ namespace MassTransit.Middleware
     using System;
     using System.Diagnostics;
     using System.Threading.Tasks;
-    using Logging;
+    using Diagnostics;
```

Comment:
Yeah, not a fan, can't this just all be in the Logging namespace? Way too many namespace includes.